doc/manual/book.toml

```
@@ -1,5 +1,5 @@
 [book]
-title = "Nix Reference Manual"
+title = "Nix reference manual"
```

I think book titles are generally capitalized (e.g. "The Rust Programming Language").
```suggestion
title = "Nix Reference Manual"
```

---

Both are legitimate (at least according to APA), but subjectively I think title case makes more sense here (ie leaving it as is right now). Perhaps because it's the title of the whole document.

---

Reverted the change since it doesn't help to get the logo merged.